projects/1.20/assets/thermal_extra/thermal_extra/lang/zh_cn.json

```
@@ -1,121 +1,338 @@
 {
+  "block.thermal_extra.abyssal_block": "深渊块",
+  "block.thermal_extra.abyssal_glass": "深渊玻璃",
+  "block.thermal_extra.advanced_refinery": "高级精炼机",
+  "block.thermal_extra.component_assembly": "部件装配机",
+  "block.thermal_extra.device_lava_gen": "积熔池",
   "block.thermal_extra.dragonsteel_block": "龙钢块",
   "block.thermal_extra.dragonsteel_glass": "龙钢玻璃",
   "block.thermal_extra.dynamo_frost": "霜冻能源炉",
   "block.thermal_extra.dynamo_frost.keyword": "暴雪 立方 雪 冰 寒冷 霜冻 冰冻 引擎 能源炉 发电机",
+  "block.thermal_extra.endothermic_dehydrator": "吸热脱水机",
+  "block.thermal_extra.fluid_mixer": "流体混合机",
+  "block.thermal_extra.nitratic_igniter": "硝化起爆机",
   "block.thermal_extra.shellite_block": "潜影合金块",
   "block.thermal_extra.shellite_glass": "潜影合金玻璃",
   "block.thermal_extra.soul_infused_block": "注灵块",
   "block.thermal_extra.soul_infused_glass": "注灵玻璃",
   "block.thermal_extra.twinite_block": "双子物质块",
   "block.thermal_extra.twinite_glass": "双子物质玻璃",
   "creative_tab.thermal_extra.augments": "热力附加丨升级",
+  "creative_tab.thermal_extra.machines": "热力附加丨机器",
   "creative_tab.thermal_extra.resources": "热力附加丨资源",
-  "fluid_type.thermal_extra.basalt": "岩石",
-  "fluid_type.thermal_extra.blitz": "狂风",
-  "fluid_type.thermal_extra.blizz": "暴雪",
+  "fluid_type.thermal_extra.abyssal": "熔融深渊",
   "fluid_type.thermal_extra.crystallized_sunflower_oil": "结晶葵花籽油",
+  "fluid_type.thermal_extra.diesel": "柴油",
   "fluid_type.thermal_extra.dragonsteel": "熔融龙钢",
-  "fluid_type.thermal_extra.icy_water": "冰水",
+  "fluid_type.thermal_extra.flux_infused_oil": "注能石油",
+  "fluid_type.thermal_extra.polyolefin": "聚烯烃",
+  "fluid_type.thermal_extra.raw_aluminum": "熔融粗铝",
+  "fluid_type.thermal_extra.raw_arcane_gold": "熔融粗神秘金",
+  "fluid_type.thermal_extra.raw_copper": "熔融粗铜",
+  "fluid_type.thermal_extra.raw_gold": "熔融粗金",
+  "fluid_type.thermal_extra.raw_iron": "熔融粗铁",
+  "fluid_type.thermal_extra.raw_lead": "熔融粗铅",
+  "fluid_type.thermal_extra.raw_nickel": "熔融粗镍",
+  "fluid_type.thermal_extra.raw_osmium": "熔融粗锇",
+  "fluid_type.thermal_extra.raw_silver": "熔融粗银",
+  "fluid_type.thermal_extra.raw_tin": "熔融粗锡",
+  "fluid_type.thermal_extra.raw_uranium": "熔融粗铀",
+  "fluid_type.thermal_extra.raw_zinc": "熔融粗锌",
   "fluid_type.thermal_extra.refined_sunflower_oil": "精炼葵花籽油",
   "fluid_type.thermal_extra.shellite": "熔融潜影合金",
   "fluid_type.thermal_extra.soul_infused": "熔融注灵",
   "fluid_type.thermal_extra.sunflower_oil": "葵花籽油",
   "fluid_type.thermal_extra.twinite": "熔融双子物质",
   "info.thermal_extra.dynamo_frost.0": "利用寒冷物品吸收的热能来产出RF。",
+  "info.thermal_extra.endothermic_dehydrator.0": "将流体蒸发为物品。",
   "info.thermal_extra.item_filter": "高级物品过滤",
+  "info.thermal_extra.lava_gen.0": "积熔池会在相邻两个及以上的熔岩源时生成熔岩。",
+  "info.thermal_extra.lava_gen_nether.0": "在下界中生成更多的熔岩。",
+  "info.thermal_extra.machine_fluid_mixer.0": "将流体混合在一起以产出其他流体。",
+  "info.thermal_extra.machine_metal_infuser.0": "通过将金属制品放入流体中以将其转化为另一种金属制品。",
```

Mix fluids together to create other fluids
Convert a metallic item into another by putting it in a fluid.
```suggestion
  "info.thermal_extra.machine_fluid_mixer.0": "将其他流体混合成一种新流体",
  "info.thermal_extra.machine_metal_infuser.0": "通过将金属制品放入流体中以将其转化为另一种金属制品。",
```
我觉得这样会更好